packages/mc-scripts/package.json

```diff
@@ -59,7 +59,7 @@
     "postcss": "7.0.17",
     "postcss-color-mod-function": "3.0.3",
     "postcss-custom-media": "7.0.8",
-    "postcss-custom-properties": "8.0.1",
+    "postcss-custom-properties": "8.0.9",
```

Comment:
this version difference wasted me so much time 😠

Reply:
What was the problem with it?

Reply:
it transformed
```css
padding: var(--spacing-4) var(--spacing-8);
```

into 

```css
padding: 4px8px;
```

Reply:
Which lead to a bunch of lint errors. It removes spaces when you have multiple vars beside each other